tests/gold_tests/continuations/double.test.py

```diff
@@ -50,11 +49,11 @@
     'map http://127.0.0.1:{0} http://127.0.0.1:{1}'.format(ts.Variables.port, server.Variables.Port)
 )
 
-numberOfRequests = randint(1000, 1500)
+numberOfRequests = 10
```

Comment:
Do we need this variable?

Reply:
I think it makes the intention of the loop more clear. or `for ((i=0; i<10; i++))` might be missed that 10 is the number of requests